docs/pages/guides/permissions.mdx

```diff
@@ -84,7 +84,7 @@ Many of these properties are also directly configurable using the [config plugin
 ```
 
 - Changes to the **Info.plist** cannot be updated over-the-air, they will only be deployed when you submit a new native binary, eg: with [`eas build`](/build/introduction).
-- Apple's official [permission message recommendations](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/requesting-permission/).
+- Apple's official [permission message recommendations](https://developer.apple.com/design/human-interface-guidelines/privacy).
```

Comment:
```suggestion
- Apple's official [permission message recommendations](https://developer.apple.com/design/human-interface-guidelines/privacy#Requesting-permission).
```

Thank your for catching that @phasza! Let's update the link to the specific section about requesting permissions.